coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -5226,3 +5226,104 @@ def roll(context, node):
     x = mb.gather(x=flatten, indices=indices, name=node.name)
     x = mb.reshape(x=x, shape=shape, name=node.name)
     context.add(x)
+
+@register_torch_op
+def im2col(context, node):
+    """
+    This op is used by torch.nn.Unfold.
+
+    Reference Implementation based on:
+        https://stackoverflow.com/questions/30109068/implement-matlabs-im2col-sliding-in-python
+
+    assert len(input.shape) == 4
```

Comment:
I probably would not include this commented out code. I don't think it's particularly helpful in understanding what's going on. A high level description in text, or simplified pseudo code, would be a lot more useful.

Reply:
Good point. Removed those commented out code and those comments in the code should be good enough.